src/it/java/io/ecocode/java/integration/tests/profile/ProfileMetadata.java

```diff
@@ -0,0 +1,42 @@
+package io.ecocode.java.integration.tests.profile;
+
+import java.util.List;
+
+public class ProfileMetadata {
+	private String name;
+	private String language;
+	private List<String> ruleKeys;
+
+	public String getName() {
```

Comment:
maybe put "lombok" framework ?

Reply:
Lombok can make it more complex for other contributors to configure some IDEs. I prefer to avoid it.